client_test.go

```diff
@@ -32,7 +32,7 @@ func TestNewClient_Defaults(t *testing.T) {
 func TestNewClient_Customized(t *testing.T) {
 	token := "deadbeef"
 	altUserAgentString := "totally-different-thing"
-	altBaseURLString := "https://api.librato.com"
+	altBaseURLString := "https://metrics-api.appoptics.com"
```

Comment:
Is this preferred? Should this be the `defaultBaseURL` in client.go as well?

Reply:
I think they both work, but I was just trying to preserve the point of this test, which was to try changing to an alternative URL and then test the non-default URL works. Other clients like the [Ruby client use api.appoptics.com](https://github.com/appoptics/appoptics-api-ruby/blob/23fe88a069efeefad3a32a7a87762238e6a77684/lib/appoptics/metrics/connection.rb#L13).